pledge-collateral.md

```diff
@@ -0,0 +1,63 @@
+# Pledge Collateral
+
+Filecoin includes a concept of "Pledge Collateral", which is FIL collateral that storage miners must lock up when participating as miners.
+
+Pledge collateral serves several functions in Filecoin. It:
+- makes it possible to slash misbehaving or slow miners
+- ensures that miners have skin in the game (for the Filecoin network as a whole)
+- increases the cost of launching a 51% attack
+
+
+## Computing Pledge Collateral
+
+The total pledge collateral across all miners is a fixed proportion of available FIL.
+Available FIL is computed as the total amount of FIL that has been mined, plus the total amount of FIL that's been vested.
+
+```go
+availableFil := minedFil + vestedFil
```

Comment:
we *could* say `- burnedFil`, otherwise over time this proportion will become effectively bigger than the percentage we set (though that will happen anyways as funds will be 'lost' through ways aside from the official burn mechanism)

Reply:
I'm fine including burnedFil. It technically becomes impossible to forecast future pledge collateral with precise accuracy, but I don't think that's an issue.